ledfx/devices/wled.py

```diff
@@ -170,7 +170,7 @@ async def async_initialize(self):
         # https://github.com/Aircoookie/WLED/pull/1944
         if wled_support_DDP(wled_build):
             _LOGGER.info(f"WLED Build Supports Sync Setting API: {wled_build}")
-            wled_sync_settings = await self.wled.get_sync_settings()
+            await self.wled.get_sync_settings()
```

Comment:
This just looks like dead code, the get should not itself change anything, so this whole block is cruft

Reply:
I am going to look at blame etc